api/hubble/account_signers.mdx

```diff
@@ -0,0 +1,10 @@
+| Column Name | Type | Description |
+| --- | --- | --- |
+| account_id | string | The address of the account. The address is the account's public key encoded in base32. All account addresses start with a 'G'.<br /><br />- Natural Key<br />- Cluster Field<br />- Required Field |
+| signer | string | The address of the account that is allowed to authorize (sign) transactions for another account. This process is called multi-sig.<br /><br />- Natural Key<br />- Required Field<br /><br />#### Notes:<br />For more information on multi-sig, read these [documments](https://developers.stellar.org/docs/encyclopedia/signatures-and-multisig?q=encyclopedia+signatures+and+multisig) |
+| weight | float64 | The numeric weight of the signer. All weights on a transaction are added up and used to determine if a transaction meets the threshold requirements to complete the transaction.<br /><br />- Natural Key<br /><br />#### Notes:<br />For example, Account A has a master key weight of 1. If Account A is attempting to submit a payment, but their threshold to complete the transaction is 2, an extra signer with a weight of "1" is required in order to complete the transaction. |
+| sponsor | string | The account address of the sponsor who is paying the reserves for this ledger entry.<br /><br />The following ledger entry types can be sponsored:<br /><br /> - accounts<br /> - account signers<br /> - claimable balances<br /> - trust lines<br /><br />#### Notes:<br />Sponsors of claimable balances are the accounts that created the balance. |
+| deleted | bool | Indicates whether the ledger entry (account, claimable balance, trust line, offer, liquidity pool) has been deleted or not. Once an entry is deleted, it cannot be recovered.<br /><br />All state tables maintain history for deleted ledger entries.<br /><br />- Required Field |
+| batch_id | string | String representation of the run id for a given DAG in Airflow. Takes the form of "scheduled__<batch_end_date>-<dag_alias>". Batch ids are unique to the batch and help with monitoring and rerun capabilities |
+| batch_run_date | datetime | The start date for the batch interval. When taken with the date in the batch_id, the date represents the interval of ledgers processed. The batch run date can be seen as a proxy of closed_at for a ledger. |
+| batch_insert_ts | timestamp | The timestamp in UTC when a batch of records was inserted into the database. This field can help identify if a batch executed in real time or as part of a backfill. The timestamp should not be used during ad hoc analysis and is useful for data engineering purposes. |
```

Comment:
What do you think about us dropping the batch fields from the docs? Since they really don't serve a purpose to the community, I wonder if we should just drop them from docs or make a one time note that `batch_id` and `batch_insert_ts` shouldn't be used